src/components/Patient/PatientHome.tsx

```
@@ -215,45 +170,114 @@
+      },
 
-  const isPatientEligibleForNewConsultation = (patientData: PatientModel) => {
-    return patientData.is_active &&
-      (!patientData?.last_consultation ||
-        patientData?.last_consultation?.discharge_date)
-      ? true
-      : false;
+      body: { allow_transfer: value },
+
+      onResponse: ({ res }) => {
+        if ((res || {}).status === 200) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)